pysteps/tests/helpers.py

```diff
@@ -193,3 +204,22 @@ def smart_assert(actual_value, expected, tolerance=None):
                                              abs=tolerance,
                                              nan_ok=True,
                                              )
+
+
+def get_invalid_mask(input_array, fillna=np.nan):
+    """
+    The the mask indicating the invalid values.
```

Comment:
Maybe good to elaborate a bit on this explanation, so that it is directly clear what this function does.